docs/articles/posts/20241203_RISE_article.md

```diff
@@ -0,0 +1,301 @@
+
+### Tool Design
+
+We used Bokeh to create our tool that can be accessed within a Jupyter Notebook. The tool was essentially an interactive scatter plot, where you could use a slider to navigate between different dimension reduction techniques.
+
+<figure markdown>
+![A gif displaying the RISE Tool interactive plot](../../images/RISE_tool/RISE_tool_recording.gif)
+<figcaption> Figure 7: A RISE tool gif, this was shown to users as preparation for how they could use the tool.</figcaption>
+</figure>
+
+Points were coloured based on the model's prediction, and there was the ability to change the shape of each point based on the human assigned labels. When you hovered over a point the image was shown. If you looked at the right had side, you could see a selection of images based on the cluster you highlighted. 
+
+This tool is available on GitHub, and you can try it out yourself [here](https://github.com/nhsengland/RISE_Tool_V1).
+
+Once our tool was created, we asked a group of assurers to trial it. Two respondents used human assigned labels within the tool, whilst three did not. Each assurer was asked to identify potential risks with the AI classifier, flag which images demonstrated said risks, asked whether they would want to generate additional test data and also comment on the useability of the tool. We'll summarise the responses from each user below.
+
+### Did this tool highlight any potential risks with the AI Classifier?
+
+Colleagues using the tool with and without labels were able to identify potential risks. When no labels were included, images of wildlife were misclassified, the image of a cat holding a tennis ball was misclassified, and some users found occasions where images containing no animals were predicted as dogs. One user pointed out the two images seen in Figure 8, which were predicted differently despite looking very similar.
+
+<figure markdown>
+![Two images of lion-like creatures, each predicted differently.](../../images/RISE_tool/Figure_8.png)
+<figcaption>Figure 8: Two images that appear very similar but were classified differently by the model. This example was found by a user using the tool.</figcaption>
+</figure>
+
+When labels were included, users were similarly successful, finding many of the same examples of those mentioned above.
+
+### Would you want to generate any additional images for testing?
+
+When labels were not included, users wanted to see more kittens, puppies and cubs, animals with their eyes closed, animals in action and more wildlife images. One user specified it would be useful to find images that are close to identical but with key features changed – this might be easier to implement now due to the recent release of '[Add it](https://research.nvidia.com/labs/par/addit/)'.  With labels, users wanted to look further into domestic cats.
+
+### Did you find this tool useful and are there any improvements you’d like to see?

```

Comment:
do you need to give this much detail on the user feedback in a blog? i think just a couple of bullet points of feedback and what your actions from it are going to be would be more appropriate for this format

Reply:
cut down significantly